CRM/Core/Form.php

```diff
@@ -1151,7 +1151,7 @@ public function toSmarty() {
    *
    * If renderer is not set create one and initialize it.
    *
-   * @return object
+   * @return self
```

Comment:
It's just a docblock but the object returned is CRM_Core_Form_Renderer which is a HTML_QuickForm_Renderer_ArraySmarty not a CRM_Core_Form.

Reply:
@demeritcowboy hmm - yeah - in the case where I hit it I wanted to know the traits functions - but I guess the form also has those so that would still work